html-syntax/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>HTML Syntax</title>
+
+</head>
+
+
+<body>
+  <img src="/html-syntax/mudkip.png" width="100" height="100">
+  <h1>Mudkip</h1>
+  <p>© Pokemon,® Nintendo</p>
```

Comment:
Please use HTML entities rather than the character itself for Copyright and Registered Trademark